store/auth.js

```diff
@@ -37,5 +44,33 @@ export const actions = {
   async logout({ commit }) {
     await Auth.signOut();
     commit("set", null);
+  },
+
+  async getApiToken({ commit }) {
+    let clientId = ''
+    let clientSecret = ''
+    const AUTHORIZATION_KEY = Buffer.from(`${clientId}:${clientSecret}`).toString('base64')
+
+    let axiosConfig = {
+      method: 'post',
+      url: 'https://ethiopia-covid19.auth.us-east-2.amazoncognito.com/oauth2/token',
```

Comment:
Could we have this as a parameter

Reply:
Yeah, I could update these to use the params/methods.